cosmo_tester/test_suites/ha/ha_cluster_scenarios_test.py

```diff
@@ -19,6 +19,7 @@
 from cosmo_tester.framework.examples.hello_world import HelloWorldExample
 from cosmo_tester.framework.cluster import CloudifyCluster
 from .ha_helper import HighAvailabilityHelper as ha_helper
+from . import community_skip
```

Comment:
Hmm... skip_community vs community_skip... no reason. You're right though, skip_<reason> is probably better, I'll tweak it.

I think I should be able to skip the entire module too, yes- I'll try it.